### PR TITLE
feat(docs): add generated openapi docs

### PR DESCRIPTION
**Pull request by lahabana**
### Checklist prior to review

- [x] [Link to relevant issue][1] as well as docs and UI issues --
- [x] This will not break child repos: it doesn't hardcode values (.e.g "kumahq" as a image registry) and it will work on Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [x] Tests (Unit test, E2E tests, manual test on universal and k8s) --
- [x] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [x] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? ([this](https://github.com/kumahq/kuma/actions/workflows/auto-backport.yaml) GH action will add "backport" label based on these [file globs](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L6), if you want to prevent it from adding the "backport" label use [no-backport-autolabel](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L8) label) --
- [x] Do you need to explicitly set a [`> Changelog:` entry here](../blob/master/CONTRIBUTING.md#submitting-a-patch) or add a `ci/` label to run fewer/more tests?

[1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### CODE REVIEW

**Comment by lahabana** [comment]
!!!! This needs an actual release of the image once: https://github.com/kumahq/ci-tools/pull/31 is merged
I've also provided the connection point to enable entreprise to use it too

File: api/openapi/specs/global_insight.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ paths:
   /global-insight:
     get:
       summary: Get Global Insight
+      tags: ["GlobalInsights"]
```
Comment (Automaat) [inline api/openapi/specs/global_insight.yaml:10]:
```suggestion
      tags: ["GlobalInsight"]
```

File: mk/docs.mk [hunk 2]
```diff
@@ -47,3 +47,21 @@ docs/generated/raw:
...
+OAPI_TMP_DIR ?= $(BUILD_DIR)/oapitmp
+API_DIRS="$(TOP)/api/openapi/specs:base"
+
+.PHONY: docs/generated/openapi.yaml
+docs/generated/openapi.yaml:
+	rm -rf $(OAPI_TMP_DIR)
+	mkdir -p $(dir $@)
+	mkdir -p $(OAPI_TMP_DIR)
+	for i in $(API_DIRS); do mkdir -p $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); cp -r $$(echo $${i} | cut -d: -f1) $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); done
+	mkdir -p $(OAPI_TMP_DIR)/policies
```
Comment (bartsmykla) [inline mk/docs.mk:60]: nit
```suggestion
	mkdir -p $(OAPI_TMP_DIR)/policies
	for i in $(API_DIRS); do mkdir -p $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); cp -r $$(echo $${i} | cut -d: -f1) $(OAPI_TMP_DIR)/$$(echo $${i} | cut -d: -f2); done
```

**Review by bartsmykla — APPROVED** [review]
One nit but LGTM overall.

**Review by Automaat — APPROVED** [review]

**Pull request merged**